### PR TITLE
Remove old Flash version checks

### PR DESCRIPTION
**Pull request by G-flat**
The Flash version checks currently in the repo are for checking whether Flash is of about version 10x or 11x, which is far behind the version we're using now (about version 32x). These checks are removed.

### CODE REVIEW

**Comment by G-flat** [comment]
A few other things that could possibly be done to refactor the code base further would be to keep just the `gc()` and `setClipboard()` functions in SystemUtil.as, with everything else in that file removed.

File: src/popups/PopupOptions.as [hunk 1]
```diff
@@ -1019,6 +999,13 @@ package popups
                 box.addChild(gameNoteColorTitle);
                 yOff += 24;
 
+                // Create ComboBox Data
```
Comment (Zageron) [inline src/popups/PopupOptions.as:1002]: Generally making the same change in two different branches is risky. Better bet would be to either not include the change or base your branch off of your feature branch, rebasing it after the previous submission.

Comment (G-flat) [reply src/popups/PopupOptions.as]: I see. I have removed the changes from my previous PR in this one.

Comment (Zageron) [reply src/popups/PopupOptions.as]: Sweet! This also helps for if this PR gets submitted before the other one, for whatever reason.

**Pull request merged**